kotlinx-coroutines-core/common/src/CoroutineDispatcher.kt

```diff
@@ -65,37 +65,76 @@ public abstract class CoroutineDispatcher :
 
     /**
      * Creates a view of the current dispatcher that limits the parallelism to the given [value][parallelism].
-     * The resulting view uses the original dispatcher for execution, but with the guarantee that
+     * The resulting view uses the original dispatcher for execution but with the guarantee that
      * no more than [parallelism] coroutines are executed at the same time.
      *
      * This method does not impose restrictions on the number of views or the total sum of parallelism values,
-     * each view controls its own parallelism independently with the guarantee that the effective parallelism
+     * each view controls its own parallelism independently with the guarantee that effective parallelism
```

Comment:
Why? I think the article is needed here.

Reply:
That's an omission, I've tried to rephrase this part multiple times and failed to find a better option